lms/templates/main.html

```diff
@@ -85,30 +92,7 @@
 
   <%block name="headextra"/>
 
-<%
-  if theme_enabled() and not is_microsite():
-    header_extra_file = 'theme-head-extra.html'
-    header_file = 'theme-header.html'
-    google_analytics_file = 'theme-google-analytics.html'
```

Comment:
This change drops support for `theme-google-analytics.html` in favor of `google_analytics.html`. This should be mentioned in the README (or support restored).